include/pprof/ddprof_pprof.hpp

```diff
@@ -5,19 +5,24 @@
 
 #pragma once
 
+extern "C" {
 #include "ddprof_context.h"
 #include "ddprof_defs.h"
 #include "ddres_def.h"
 #include "perf_watcher.h"
 #include "unwind_output.h"
+}
+#include "tags.hpp"
 
 typedef struct ddprof_ffi_Profile ddprof_ffi_Profile;
 typedef struct SymbolHdr SymbolHdr;
 
 typedef struct DDProfPProf {
+  DDProfPProf() : _profile(nullptr), _nb_values(0) {}
```

Comment:
[Minor] You should prefer in-class initializers to member initializers in constructors (https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#Rc-default)